paddle/fluid/framework/details/all_reduce_op_handle.cc

```diff
@@ -52,7 +61,185 @@ AllReduceOpHandle::AllReduceOpHandle(ir::Node *node,
     : OpHandleBase(node), local_scopes_(local_scopes), places_(places) {}
 #endif
 
+#if defined(PADDLE_WITH_CUDA) && !defined(_WIN32)
+void AllReduceOpHandle::_RunImplEncoded() {
```

Comment:
seems no under scroll started function names in google style?

Reply:
Done.Thanks!